lib/base/timer.cpp

```diff
@@ -295,11 +299,11 @@ void Timer::TimerThreadProc()
 		auto it = idx.begin();
 		Timer *timer = *it;
 
-		double wait = timer->m_Next - Utility::GetTime();
+		auto next (ch::system_clock::from_time_t(0) + ch::duration<double>(timer->m_Next));
```

Comment:
Why not use the constructor of `time_point` that takes a duration after the clock's epoch? See (2) on https://en.cppreference.com/w/cpp/chrono/time_point/time_point